Makefile

```diff
@@ -41,14 +41,17 @@ APP_VER		:= $(shell git describe 2> /dev/null || echo "unknown")
 REGISTRY_APP_NAME		:= a8registry
 CONTROLLER_APP_NAME		:= a8controller
 SIDECAR_APP_NAME		:= a8sidecar
+K8SRULES_APP_NAME		:= a8k8srulescontroller
 
```

Comment:
Do we need a separate binary? Why can't we merge this into the current A8 controller ?